wolfssl/wolfcrypt/hash.h

```diff
@@ -119,6 +119,7 @@ typedef union {
 #ifdef WOLFSSL_SM3
     wc_Sm3 sm3;
 #endif
+    WOLF_AGG_DUMMY_MEMBER;
```

Comment:
Are we going to use wc_Hashes without a digest defined?
Does it matter if the union has a 1 byte array field when it isn't used?